File: net.tenteCsharp/src_code/heaterManagement/IHeaterGUI.java
package heaterManagement;
import heaterManagement.HeaterManagement.*;

public interface IHeaterGUI{
	
	public void setTemperature(String heaterId,float value);	
	public void programSwitchOn(String heaterId,float value,String time);	
	public void programSwitchOff(String heaterId,String time);

}

File: trunk/pseudoCodeGeneratorElio/src-gen/heaterManagement/IHeaterGUI.cs
using System;
using System.Collections;
using System.Text;

namespace SmartHome
{
	public interface IHeaterGUI
	{
		void setTemperature(String heaterId, float value);
		void programSwitchOn(String heaterId, float value, String time);
		void programSwitchOff(String heaterId, String time);

	}
}
